package.json

```diff
@@ -37,7 +37,9 @@
     "coveralls": "^3.0.2",
     "eslint-plugin-import": "^2.14.0",
     "eslint-plugin-jsx-a11y": "^6.1.1",
-    "eslint-plugin-react": "^7.11.0"
+    "eslint-plugin-react": "^7.11.0",
+    "node-sass": "^4.10.0",
+    "sass-loader": "^7.1.0"
```

Comment:
i don't think we need `sass-loader` anymore, with `create-react-app` version 2.x sass is supported and only need `node-sass`